src/tennisSimulator/Menu.java

```diff
@@ -0,0 +1,94 @@
+package tennisSimulator;
+
+import java.util.Scanner;
+
+public class Menu {
+
+    private static final PlayersDatabase playersDatabase = new PlayersDatabase();
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public void openTennisSimulatorMenu() {
+        boolean quit = false;
+        System.out.println("WELCOME TO TENNIS SIMULATOR\n");
+        printMenu();
+        while (!quit) {
+            int choice = scanner.nextInt();
+            scanner.nextLine();
+            switch (choice) {
+                case 0:
+                    printMenu();
+                    break;
+                case 1:
+                    playersDatabase.print();
+                    break;
+                case 2:
+                    addPlayerToDatabase();
+                    break;
+                case 3:
+                    quit = true;
+                    System.out.println("Exiting simulator...");
+                    break;
+                default:
+                    System.out.println("Incorrect input. Please choose one of available options.\n");
+                    printMenu();
+                    break;
+            }
+        }
+    }
+
+    private void printMenu() {
+        System.out.println("Select:\n" +
+                "0 - show options;\n" +
+                "1 - show players database;\n" +
+                "2 - add player to database;\n" +
+                "3 - exit simulator;");
+    }
+
+    private void addPlayerToDatabase() {
+        System.out.print("Enter a full player name: ");
+        String name = scanner.nextLine();
+
+        if (!isFullName(name)) {
+            System.out.println("Addition failed. Please enter full player name.");
+            return;
+        }
+
+        System.out.print("Enter a gender (man/woman): ");
+        String gender = scanner.nextLine().toLowerCase();
+
+        if (!isValidGender(gender)) {
+            System.out.println("Addition failed. Please enter a valid gender.");
+            return;
+        }
+
+        resolveGender(gender);
+        int rating = scanner.nextInt();
+        if (!isValidRating(rating)) {
+            System.out.println("Addition failed. Please enter a valid rating.");
+            return;
+        }
+
+        playersDatabase.addPlayer(new Player(name, gender, rating));
+
+    }
+
+    private boolean isValidGender(String gender) {
+        return gender.equalsIgnoreCase(Player.MAN) || gender.equalsIgnoreCase(Player.WOMAN);
+    }
+
+    private boolean isFullName(String name) {
+        return name.contains(" ");
+    }
+
+    private boolean isValidRating(int rating) {
+        return rating >= 0;
+    }
+
+    private void resolveGender(String gender) {
```

Comment:
Better but still problem exists. 
If I wouldn't check the implementation of this method and would trust the name I would believe that it should probably return me a resolved gender, but now it asks user to enter a rating which is not related to gender.
Maybe even better idea would be to name it resolveOrganization which would return the organization you need and then from the place where you invoke this method you could ask for user input of rating.